src/convertToObject.js

```diff
@@ -6,7 +6,12 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((item) => item.includes(':'))
+    .map((item) => item.split(':'))
+    .map(([property, value]) => [property.trim(), value.trim()])
+    .reduce((acc, [property, value]) => ({ ...acc, [property]: value }), {});
```

Comment:
The chained methods should have the same indentation for better readability. Each method should start on a new line and be indented to align with the others.